api-reference/beta/api/directoryobject_validateproperties.md

```diff
@@ -0,0 +1,145 @@
+# directoryObject: validateProperties
+
+Validate an Office 365 group's display name or mail nickname complies with naming policies.  Clients can use the API to determine if a display name or mail nickname is valid before trying to create an Office 365 group. 
+
+The following validations are performed for the display name and mail nickname properties: 
+1. Validate the prefix and suffix naming policy
+2. Validate the custom banned words policy
+3. Validate the mail nickname is unique
```

Comment:
Maybe we should clarify that it only checks uniqueness against groups with groupType "Unified"? I don't know if we already clarify that in the Group creation docs